.github/workflows/sync-config-schema.yaml

```diff
@@ -30,20 +31,33 @@ jobs:
             final_tag=${{ github.event.release.tag_name }}
           fi
           echo "release_tag=${final_tag}" >> "$GITHUB_OUTPUT"
-          if [[ ${final_tag} == *"-beta."* || ${final_tag} == *"-alpha."* ]]; then
-            echo "update_main=true" >> "$GITHUB_OUTPUT"
+          if [[ ${final_tag} == *"-alpha."* ]]; then
+            echo "is_alpha_version=true" >> "$GITHUB_OUTPUT"
+            echo "is_stable_version=false" >> "$GITHUB_OUTPUT"
           else
-            echo "update_main=false" >> "$GITHUB_OUTPUT"
+            echo "is_alpha_version=false" >> "$GITHUB_OUTPUT"
           fi
+          if [[ ${final_tag} == *"-beta."* || ${final_tag} == *"-rc"* ]]; then
+            echo "is_stable_version=false" >> "$GITHUB_OUTPUT"
+          else
+            echo "is_stable_version=true" >> "$GITHUB_OUTPUT"
+          fi
+
+      - name: Skip sync on alpha
+        if: ${{ steps.release.outputs.is_alpha_version == 'true' }}
+        env:
+          RELEASE_TAG: ${{ steps.release.outputs.release_tag }}
+        run: echo "skipping sync because release ${RELEASE_TAG} is alpha"
 
       - name: Checkout repo
-        if: ${{ steps.release.outputs.update_main == 'false' }}
+        if: ${{ steps.release.outputs.is_alpha_version == 'false' }}
```

Comment:
`steps.release.outputs.is_stable_version == 'false' && && steps.release.outputs.is_alpha_version == 'false' `?

Reply:
no, we want to do this on betas, RC and full versions (see my comment below)